tsconfig.json

```diff
@@ -15,7 +15,8 @@
 
   },
   "include": [
-    "./src/**/*.ts"
+    "./src/**/*.ts",
+    "global.d.ts"
```

Comment:
And why is "global.d.ts" added?

Reply:
Good points! The `global.d.ts` file is purely for typing the global mock variables. Regarding the production build, the declaration file don’t actually emit any JavaScript or impact runtime code, so they won’t alter the production bundle. Still, I've moved it into the jest_config file to keep things tidy and avoid confusion.